charts/coturn/README.md

```diff
@@ -1,10 +1,11 @@
 **Warning**: this chart is currently considered alpha. Use at your own risk!
 
-This chart deploys [coturn](https://github.com/coturn/coturn), a STUN and TURN
+This chart deploys [coturn](https://github.com/wireapp/coturn), a STUN and TURN
```

Comment:
IMHO, we should still link to upstream here, and maybe additional to our branch with the changes on top.

We intend to upstream all this one way or another, right?

Reply:
> IMHO, we should still link to upstream here, and maybe additional to our branch with the changes on top.

That's maybe not a bad idea, I could be convinced of that.

> We intend to upstream all this one way or another, right?

I'm not sure that all of our changes are upstreamable (especially some of the authentication changes), and I don't otherwise have the time just now to run all our changes through the pipeline of proposing them to upstream.

Reply:
I'm not saying we should upstream this now ;-)

I only think we should primarily link to upstream, and /then/ to our local branch with changes on top to convey this is some changes on top of upstream, but not its own project.

Reply:
Fixed in [d7a8a72](https://github.com/wireapp/wire-server/pull/2620/commits/d7a8a72b122733da98a8a23b2b6403f489ba6b02).